pkg/crc/constants/constants.go

```diff
@@ -123,26 +122,26 @@ func EnsureBaseDirectoriesExist() error {
 	return os.MkdirAll(CrcBaseDir, 0750)
 }
 
-// IsBundleEmbedded returns true if the executable was compiled to contain the bundle
-func BundleEmbedded() bool {
-	executablePath, err := os.Executable()
-	if err != nil {
-		return false
-	}
-	extractor, err := binappend.MakeExtractor(executablePath)
+// BundlePresent returns true and path of the bundle
+// if the bundle available in user's home directory.
+func BundlePresent() (bool, string) {
+	var bundlePath string
+	var present bool
+	err := filepath.Walk(GetHomeDir(),
```

Comment:
I don't think we want to do this. Either the bundle is present in ~/.crc/cache, or it's in the same directory as the crc executable, or the user will have to specify it with `-b`.

Reply:
@cfergeau I am doing it to retain the existing workflow as much as possible, do you think it might have any kind of side effect except the time taken by this logic if `HOMEDIR` is huge?

Reply:
on our meeting I said exactly the same as Christophe. Anything else is guess work and should be avoided.

Reply:
if the bundle is NOT in the current dir, the user likely moved the binary out of place (without considering the bundle), so this ia a user error.

Fail early ... and fail with a clear error/instruction.